### PR TITLE
added changes for VA close end point

### PR DESCRIPTION
**Pull request by neera11**

### CODE REVIEW

File: razorpay/resources/virtual_account.py [hunk 1]
```diff
@@ -53,9 +53,9 @@ def close(self, virtual_account_id, data={}, **kwargs):
             virtual_account_id :
                 Id for which Virtual Account objects has to be Closed
         """
-        url = "{}/{}".format(self.base_url, virtual_account_id)
-        data['status'] = 'closed'
-        return self.patch_url(url, data, **kwargs)
+        url = "{}/{}/close".format(self.base_url, virtual_account_id)
+        # data['status'] = 'closed'
```
Comment (ritesh-commits) [inline razorpay/resources/virtual_account.py:57]: let's remove this comment

**Review by ritesh-commits — APPROVED** [review]
lgtm

**Review by neera11 — COMMENTED** [review]
removed comment

**Comment by captn3m0** [comment]
Ran tests locally, failing:

```
======================================================================
ERROR: test_virtual_accounts_close (tests.test_client_virtual_account.TestClientVirtualAccount)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "<string>", line 3, in wrapper
  File "/home/nemo/projects/work/razorpay/razorpay-python/tests/test_client_virtual_account.py", line 83, in test_virtual_accounts_close
    response = self.client.virtual_account.close(
  File "/home/nemo/projects/work/razorpay/razorpay-python/razorpay/resources/virtual_account.py", line 57, in close
    return self.post_url(url, data, **kwargs)
  File "/home/nemo/projects/work/razorpay/razorpay-python/razorpay/resources/base.py", line 20, in post_url
    return self.client.post(url, data, **kwargs)
  File "/home/nemo/projects/work/razorpay/razorpay-python/razorpay/client.py", line 155, in post
    return self.request('post', path, data=data, **options)
  File "/home/nemo/projects/work/razorpay/razorpay-python/razorpay/client.py", line 119, in request
    response = getattr(self.session, method)(url, auth=self.auth,
  File "/home/nemo/.local/lib/python3.9/site-packages/requests/sessions.py", line 590, in post
    return self.request('POST', url, data=data, json=json, **kwargs)
  File "/home/nemo/.local/lib/python3.9/site-packages/requests/sessions.py", line 542, in request
    resp = self.send(prep, **send_kwargs)
  File "/home/nemo/.local/lib/python3.9/site-packages/requests/sessions.py", line 655, in send
    r = adapter.send(request, **kwargs)
  File "/home/nemo/.local/lib/python3.9/site-packages/responses/__init__.py", line 765, in unbound_on_send
    return self._on_request(adapter, request, *a, **kwargs)
  File "/home/nemo/.local/lib/python3.9/site-packages/responses/__init__.py", line 745, in _on_request
    raise response
requests.exceptions.ConnectionError: Connection refused by Responses - the call doesn't match any registered mock.

Request:
- POST https://api.razorpay.com/v1/virtual_accounts/va_4xbQrmEoA5WJ0G/close

Available matches:
- PATCH https://api.razorpay.com/v1/virtual_accounts/va_4xbQrmEoA5WJ0G Method does not match


----------------------------------------------------------------------
```

Please fix. I've re-configured travis so the next commit should trigger a build correctly.

**Pull request merged**